libs/servo/servo.ts

```diff
@@ -36,11 +38,15 @@ namespace servos {
         //% group="Positional"
         setAngle(degrees: number) {
             degrees = this.clampDegrees(degrees);
-            this.internalSetAngle(degrees);
+            this._angle = this.internalSetAngle(degrees);
         }
 
-        protected internalSetAngle(angle: number): void {
+        get angle() {
+            return this._angle || 90;
+        }
 
+        protected internalSetAngle(angle: number): number {
+            return 0;
```

Comment:
Why is this always 0?

Reply:
overriden on base class, abstract breaks our compiler